src/crawlee/events/_types.py

```diff
@@ -79,7 +82,23 @@ class EventExitData(BaseModel):
     model_config = ConfigDict(populate_by_name=True)
 
 
-EventData = Union[EventPersistStateData, EventSystemInfoData, EventMigratingData, EventAbortingData, EventExitData]
+@docs_group('Event payloads')
+class EventCrawlerStatusData(BaseModel):
+    """Data for the crawler status event."""
+
+    model_config = ConfigDict(populate_by_name=True)
+
+    message: str
```

Comment:
May I suggest adding something to distinguish crawlers for cases where there are multiple instances in a single run? If there's nothing more meaningful, `id(crawler)` should be fine.